PR: https://github.com/odoo/odoo/pull/

From: 8a5360bef7427b70364d503837a5b825ee5ac9e4
From: Pratima Gupta
Date: 2017-01-31 13:09:58

Structural Changes: 7
Total Changes: 8

[IMP] hr_payroll: change the config settings view according to new design

================================= pseudo patch: =================================

--- a/addons/hr_payroll/models/res_config.py
+++ b/addons/hr_payroll/models/res_config.py
@@ -8,5 +8,9 @@ class HrPayrollConfigSettings(models.TransientModel):
     _name = 'hr.payroll.config.settings'
     _inherit = 'res.config.settings'
 
-    module_hr_payroll_account = fields.Boolean(string='Link your payroll to accounting system',
-        help="Create journal entries from payslips")
+    module_hr_timesheet = fields.Boolean(string='Timesheets')
+    module_website_sign = fields.Boolean(string='Online Signatures')
+    module_account_accountant = fields.Boolean(string='Account Accountant')
+    module_l10n_fr_hr_payroll = fields.Boolean(string='French Payroll')
+    module_l10n_be_hr_payroll = fields.Boolean(string='Belgium Payroll')
+    module_l10n_in_hr_payroll = fields.Boolean(string='Indian Payroll')
